ci-operator/jobs/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-master-periodics.yaml

```diff
@@ -0,0 +1,76 @@
+periodics:
+- agent: kubernetes
+  cluster: build01
+  cron: 0 */23 * * *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: master
+    org: openshift-psap
+    repo: ci-artifacts
+  labels:
+    ci-operator.openshift.io/prowgen-controlled: "true"
+    ci-operator.openshift.io/variant: nfd
+    job-release: "4.8"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-psap-ci-artifacts-master-nfd-e2e-master
+  spec:
```

Comment:
slack reporting missing here

Reply:
(but the test will disappear from here if you follow the suggestion to remove periodics from `master`)